src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+public class SortCommand extends Command {
```

Comment:
Remember to do the header comments for your classes as well, so that to have an overview of what the class will be doing.